src/components/MaterialUI/DataDisplay/Table.js

```diff
@@ -93,7 +93,7 @@ export const useStyles = makeStyles(theme => ({
 function headersAreIdentical(prevHeaders, nextHeaders) {
 	prevHeaders.forEach((prevHeader, index) => {
 		let prevSortOptions = prevHeader.cellElement.props.columnDefinition.sortOptions;
-		let nextSortOptions = nextHeaders[index].cellElement.props.columnDefinition.sortOptions;
+		let nextSortOptions = nextHeaders[index]?.cellElement.props.columnDefinition.sortOptions;
```

Comment:
I think we should just take sortOptions for "nextHeader" in the same way as we do it for "prevHeader"? So, nextHeaders.cellElement.props.columnDefintion.sortOptions.

Reply:
But it's array, We need index. And I added ? sign because in my story I have a case when in nextHeaders we have different amount of columns

Reply:
nvm, I missed the thing how you are comparing it